setup.py

```diff
@@ -356,6 +363,9 @@
     'obspy.plugin.inventory.CSS': [
         'writeFormat = obspy.io.css.station:_write_css',
         ],
+    'obspy.plugin.inventory.SEISAN': [
+        'writeFormat = obspy.io.nordic.core:inventory_to_seisan',
```

Comment:
👍 Have moved this from here and made it a gist, so if anyone comes looking they can find it [here.](https://gist.github.com/calum-chamberlain/ca84185a324dc42d631808d94e4ad0c6)